test_framework/test_framework.py

```diff
@@ -100,7 +100,7 @@ def __init__(self):
         self.supports_cli = False
         self.bind_to_localhost_only = True
         self.set_test_params()
-        self.parse_args()
+        # self.parse_args()
```

Comment:
Yep I had to do that too. I have no idea why is that happening.

Reply:
It's because jupyter inserts its own command line args, which then fail parsing by the test framework:

```
import sys
print(sys.argv)

['.venv/lib/python3.7/site-packages/ipykernel_launcher.py', '-f', '/home/john/.local/share/jupyter/runtime/kernel-6b886fa3-edb3-4e87-bef5-1032ce2ebf84.json']
```